components/chef-ui-library/src/molecules/chef-modal/chef-modal.tsx

```diff
@@ -115,13 +115,17 @@ export class ChefModal {
     if (visible) {
       this.prevFocusedElement = document.activeElement as HTMLElement;
 
-      // sets the focus on the modal once Angular has detected modal is open
-      const focusModal = setInterval(() => {
-        const modal = this.el.getElementsByClassName('modal').item(0) as HTMLElement;
-        modal.focus();
-
-        if (modal === document.activeElement) {
-          clearInterval(focusModal);
+      // when Angular detects the modal is open
+      // sets the focus on the close button for unlocked modals,
+      // or the div for locked modals
+      const focusElement =
+        (this.locked ? this.el.getElementsByClassName('modal').item(0) :
+        this.el.getElementsByClassName('close').item(0).firstElementChild) as HTMLElement;
+
+      const focusElementInterval = setInterval(() => {
+        focusElement.focus();
+        if (focusElement === document.activeElement) {
+          clearInterval(focusElementInterval);
         }
       }, 1);
```

Comment:
We could do this more concisely unless there is a reason to be waiting until the element gets focus (which is what the conditional checks for) ...? So with this change sometimes the close button would get outlined and sometimes not, but the initial problem is still corrected.
```suggestion
      setTimeout(() => focusElement.focus(), 1);
```

Reply:
yah idk that code was already there? i assumed it was needed...